airflow-core/src/airflow/ui/src/pages/Dag/Calendar/CalendarCell.tsx

```diff
@@ -103,5 +96,9 @@ export const CalendarCell = ({
     return cellBox;
   }
 
-  return <HoverTooltip tooltip={renderTooltip(cellData, viewMode)}>{cellBox}</HoverTooltip>;
+  return (
+    <BasicTooltip content={<CalendarTooltip cellData={cellData} viewMode={viewMode} />} placement="bottom">
```

Comment:
```suggestion
    <BasicTooltip content={<CalendarTooltip cellData={cellData} viewMode={viewMode} />}>
```

Bottom is the default.